docs/source/changelog.rst

```diff
@@ -6,13 +6,22 @@ Changelog
     * Enhancements
     * Fixes
     * Changes
+        * Remove the ability to stack transform primitives (:pr:`1119`)
     * Documentation Changes
     * Testing Changes
         * Better test case for normalizing from no time index to time index (:pr:`1113`)
 
     Thanks to the following people for contributing to this release:
     :user:`tamargrey`
 
+**Breaking Changes**
+
+    * ``ft.dfs`` will no longer build features from transform primitives where one of the 
+    inputs was also built with a transform primitve. This will make some 
```

Comment:
would "where on of the inputs was a Transform feature, a GroupByTransform feature, or a Direct Feature of a Transform / GroupByTransform feature" be an accurate description?

Reply:
yes, that would be the most accurate way to say this

Reply:
ended up adding this in